File: ignored-test/CSharpXUnit/CSharpXUnit.Tests/Tests/Configuration_IsValid.cs
using Xunit;

namespace DefaultNamespace
{
    public class AutoMapperConfigurationTests
    {
        [Fact(Skip = "")]
        public void Configuration_IsValid()
        {
            {
                var config = new MapperConfiguration(cfg => cfg.AddProfile(new DefaultProfile()));
                config.AssertConfigurationIsValid();
            }
        }
    }
}

File: refactored/ignored-test/java-junit/Configuration_IsValid.java
import org.junit.Test;
import org.junit.Ignore;
import static org.junit.Assert.*;

public class AutoMapperConfigurationTests {

    @Test
    // @Ignore("")
    public void configuration_IsValid() {
        MapperConfiguration config = new MapperConfiguration(cfg -> cfg.addProfile(new DefaultProfile()));
        config.assertConfigurationIsValid();
    }
}
